ansible/roles/ocp4-workload-dil-streaming/tasks/provision_dg.yaml

```diff
@@ -0,0 +1,17 @@
+---
+- name: Evaluate user{{ item }} namespace if not exists 
+  k8s:
+    api_version: v1
+    kind: Namespace
+    name: 'user{{ item }}'
+    state: present
+
+- name: Evaluate Data Grid Operator Group
+  k8s:
+    state: present
+    resource_definition: "{{ lookup('template', 'datagrid-operatorgroup.yaml.j2') }}"
+
+- name: Evaluate Data Grid Subscription
+  k8s:
+    state: present
+    resource_definition: "{{ lookup('template', 'datagrid-subscription.yaml.j2') }}"
```

Comment:
This really should have checks to ensure the operator installed successfully.
Example: https://github.com/redhat-cop/agnosticd/blob/development/ansible/roles_ocp_workloads/ocp4_workload_serverless/tasks/workload.yml#L13-L79

Also is there a risk of the operators changing? If so a Catalog Snapshot should be used to prevent environments from breaking: https://github.com/redhat-cop/agnosticd/blob/development/docs/Operator_Catalog_Snapshots.adoc

Reply:
Will be addressed in subsequent PRs. This is a break fix for a broken environment